libs/service/api_service.cpp

```diff
@@ -431,6 +432,73 @@ ebpf_verify_and_load_program(
 
 #endif
 
+_Must_inspect_result_ ebpf_result_t
+ebpf_authorize_native_module(_In_ HANDLE native_image_handle) noexcept
+{
+    EBPF_LOG_ENTRY();
+
+    ebpf_result_t result = EBPF_SUCCESS;
+    HANDLE file_mapping_handle = NULL;
+    void* file_mapping_view = nullptr;
+    size_t file_size = 0;
+    ebpf_operation_authorize_native_module_request_t request;
+    uint32_t error = ERROR_SUCCESS;
+
+    file_mapping_handle = CreateFileMappingA(native_image_handle, nullptr, PAGE_READONLY, 0, 0, nullptr);
```

Comment:
```suggestion
    file_mapping_handle = CreateFileMappingW(native_image_handle, nullptr, PAGE_READONLY, 0, 0, nullptr);
```
The last arg is null so no need to use ANSI